project/tasks/collaborate.rst

```diff
@@ -0,0 +1,93 @@
+====================
+Collaborate on Tasks
+====================
+
+Being able to collaborate virtually, but with the same effect as if people were in the same room,
+increases your productivity. It results in faster execution of processes,
+brainstorming and decision made easier while also allowing the support of outside contributions.
+
+Add followers to a project
+==========================
+
+| *Edit* the project to add followers.
+| Followers are notified of any changes: tasks sliding from one stage to another, conversations
+  taking place, etc.
+
+.. image:: media/add_followers.png
+   :align: center
+   :alt: Click on the follower icon to add followers to a project in Odoo Project
+
+.. note::
+   Following a project automatically makes you a follower of all its tasks.
+
+Add followers to a task
+=======================
+
+Within a task, click on the *Followers* icon. They get notified of any changes in the task,
+and can see the task like you do, with the description and the chatter.
```

Comment:
```suggestion
Within a task, click on the *Followers* icon. Those followers get notified for any change in the task
and can see the task like you do, with the description and the chatter.
```

Reply:
I see the point of the suggestion but I rewrote in a different way.
And to confirm a did a little research: and 'notified of' is correct (: